### PR TITLE
l10n: zh_CN v2.39.0 round 1

### PR DESCRIPTION
**Pull request by fangyi-zhou**
Signed-off-by: Fangyi Zhou <me@fangyi.io>

Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -23651,6 +23797,16 @@ msgstr "scalar reconfigure [--all | <登记>]"
 msgid "--all or <enlistment>, but not both"
 msgstr "--all 或者 <登记>，而不是两个一起"
 
+#: scalar.c
+#, c-format
+msgid "could not remove stale scalar.repo '%s'"
+msgstr "无法删除过时的 scalar.repo '%s'"
```
Comment (lilydjwg) [inline po/zh_CN.po:23803]:
```suggestion
msgstr "无法删除过期的 scalar.repo '%s'"
```

File: po/zh_CN.po [hunk 2]
```diff
@@ -23651,6 +23797,16 @@ msgstr "scalar reconfigure [--all | <登记>]"
...
+
+#: scalar.c
+#, c-format
+msgid "removing stale scalar.repo '%s'"
+msgstr "正在删除过时的 scalar.repo '%s'"
```
Comment (lilydjwg) [inline po/zh_CN.po:23808]:
```suggestion
msgstr "正在删除过期的 scalar.repo '%s'"
```

File: po/zh_CN.po [hunk 3]
```diff
@@ -13415,6 +13469,11 @@ msgstr "无法打开临时文件 %s 进行写入"
 msgid "could not close refs snapshot tempfile"
 msgstr "不能关闭引用快照临时文件"
 
+#: builtin/repack.c
+#, c-format
+msgid "could not remove stale bitmap: %s"
+msgstr "无法删除过时的位图： %s"
```
[the remark this answers is not in the document]
Comment (fangyi-zhou) [reply po/zh_CN.po:13475]: 其他地方的 stale 被翻译成了『过时』 是否要一起修正？

[the remark this answers is not in the document]
Comment (lilydjwg) [reply po/zh_CN.po:13475]: 嗯，如果决定要改的话显然所有「stale」要一起改的。

**Comment by jiangxin** [comment]
Hi @fangyi-zhou ,

This PR LGTM.  Git 2.39.0-rc2 has been released with no l10n update. Will merge this PR after you remove the draft tag.

**Pull request merged**